VERSION

```diff
@@ -1 +1 @@
-2.14.3
+3.2.4
```

Comment:
We will need to continue maintaining the helm 2.x image for clusters that use tiller to maintain existing deployments. To do this, we'll probably want a `make` target here that can iterate through a list of supported versions and build all of them. Or we branch the repo. Not sure what the best path forward is. Discussion welcome.

Reply:
Im good with either approach. We need to update off of 2.14.3 as well, there's a number of fixes since that was released.

Reply:
Wanted to create a branch helm3 .. but didn't have the permissions to do so. it would be easier to maintain considering changes to these images won't be very often.

Reply:
@kd7lxl / @drewwells , can either of you create a separate branch `helm3` ?

Reply:
Created https://github.com/infobloxopen/helm/tree/helm3